client/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "@bower_components/backbone": "jashkenas/backbone#1.1.2",
+    "@bower_components/bootstrap": "twbs/bootstrap#3.3.2",
+    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#4.3.0",
+    "@bower_components/jquery": "jquery/jquery-dist#2.1.3",
+    "@bower_components/lodash": "lodash/lodash#3.3.1",
+    "@bower_components/moment": "moment/moment#2.9.0",
+    "@bower_components/require-handlebars-plugin": "SlexAxton/require-handlebars-plugin#0.11.2",
+    "@bower_components/requirejs": "jrburke/requirejs-bower#2.1.16",
+    "@bower_components/roboto-fontface": "choffmeister/roboto-fontface-bower#0.2.7",
+    "@bower_components/underscore": "jashkenas/underscore#>=1.5.0",
+    "cjs": "^0.0.11",
+    "express": "^4.18.2"
```

Comment:
remove carats here
can add a .npmrc with [legacy deps](https://docs.npmjs.com/cli/v7/using-npm/config#legacy-peer-deps) to stop from happening in future